.gitignore

```diff
@@ -17,3 +17,4 @@ pdq/java/bazel-out/
 pdq/java/bazel-testlogs/
 .mypy_cache/
 .vscode/
+hasher-matcher-actioner/.devcontainer/devcontainer.json
```

Comment:
I see this is added here... Not sure if we want to do this... If we go this route we'll need a convention to call out when to go get a new copy, and we'll have to store the "official" team copy somewhere outside of source control. I'm pretty sure this should be left in as-is. @schatten what do you think?

Reply:
Didnt you need to change this file to set up the dev container? Change dipanjanm name to your unix name?